larray/example.py

```diff
@@ -0,0 +1,54 @@
+import os
+import larray as la
+
+EXAMPLE_FILES_DIR = os.path.dirname(__file__) + '/tests/data/'
+AVAILABLE_EXAMPLE_DATA = {
+    'demography' : EXAMPLE_FILES_DIR + 'data.h5'
+}
+
+def load_example_data(example_data=None):
+    """Load arrays used in the tutorial so that all examples in it can be reproduced.
+
+    Parameters
+    ----------
+    example_data : str
+        Example data to load. Available example datasets are:
+
+        - demography
```

Comment:
Wouldn't it be better to populate this list automatically, so that we do not have to keep it in sync?